src/components/App.svelte

```diff
@@ -1,11 +1,12 @@
 <script>
-import { me } from '../data/user'
-import { init as loadEvents } from '../data/events'
-import { init as loadMessaging } from '../data/messaging'
-import { init as loadRequests } from '../data/requests'
-import { Router } from '@roxi/routify'
+import { init as loadEvents } from 'data/events.js'
+import { init as loadMessaging } from 'data/messaging.js'
+import { init as loadRequests } from 'data/requests.js'
+import { me } from 'data/user.js'
 import { routes } from '../../.routify/routes'
+
```

Comment:
is this line break intentional?

Reply:
Yes, as I mentioned in the PR description, it separates the internal imports from the external imports

Reply:
Though if you don't like it, I would be OK with not doing this.